docs/api/vectorizer.rst

```diff
@@ -61,6 +61,41 @@ CohereTextVectorizer
    :show-inheritance:
    :members:
 
+BedrockTextVectorizer
+====================
```

Comment:
Need one more `=` to make it line up with the line length of the `BedrockTextVectorizer`. Otherwise, sphinx will have some issues there parsing